conan/tools/apple/apple.py

```diff
@@ -74,7 +74,8 @@ def apple_min_version_flag(conanfile):
 
 
 def apple_sdk_path(conanfile):
-    sdk_path = conanfile.conf["tools.apple:sdk_path"]
+    sdk_path = conanfile.conf.get("tools.apple:sdk_path")
+    # It's better to keep the default return later in this case
```

Comment:
What do you mean with this comment?

Reply:
I meant that we don't want to run the `xcrun` tool always without knowing if it was given a `sdk_path` previously via `conf`.

Reply:
I think the comment is confusing and does not make much sense outside the context of this PR, I think it would be better to remove it or extend add more detailed information of why that is important...